mail_footer_notified_partners/models/mail_followers.py

```diff
@@ -57,7 +57,7 @@ def _notify(
             ctx.update({
                 'partners_to_notify': partners_to_notify,
             })
-        return super(MailNotification, self)._notify(
+        return super(MailNotification, self.with_context(ctx))._notify(
             message_id,
             partners_to_notify=partners_to_notify,
             force_send=force_send, user_signature=user_signature, context=ctx)
```

Comment:
remove the second context here, and possibly squash your commits. If you also add 'fixes #NNN' in the commit message, the issue will be closed automatically when this is merged